Classes/Controller/MailqueueModuleController.php

```diff
@@ -92,7 +95,7 @@ public function __invoke(Message\ServerRequestInterface $request): Message\Respo
         if ($this->typo3Version->getMajorVersion() < 12) {
             return $this->renderLegacyTemplate($template, $templateVariables);
         }
-
+        $this->assertAllowedHttpMethod($request, 'GET');
```

Comment:
The assertion should happen as very first check in the action. Thus, it should be moved to line 64. Since POST and GET methods are allowed it should be changed to:

```suggestion
        $this->assertAllowedHttpMethod($request, 'GET', 'POST');
```